CHANGELOG.md

```diff
@@ -5,6 +5,14 @@
 * Improve support for tab characters, and horizontal rules.
 * The above fixes raise compliance with the CommonMark specs to 90%, and
   compliance with the GFM specs to 87%.
+* Allow the binary script to take a `--extension-set` option.
+
+  A reminder: You can run `bin/markdown.dart` from anywhere via:
+
+  ```shell
+  $ pub global activate markdown
+  $ pub global run markdown
```

Comment:
Just `markdown` should work if they have their paths set up correctly...

Reply:
Done.